File: AOJ/ITP1_re/itp_1_3_d.go
package main

import "fmt"

func main() {
	var a, b, c, cnt int
	fmt.Scan(&a, &b, &c)
	for i := a; i <= b; i++ {
		if c%i == 0 {
			cnt++
		}
	}
	fmt.Println(cnt)
}

File: AOJ/ITP1_re/itp_1_3_d.cpp
#include <bits/stdc++.h>

int main() {
  int a, b, c, cnt;
  scanf("%d %d %d", &a, &b, &c);
  for (int i = a; i <= b; i++) {
    if (c % i == 0) {
      cnt++;
    }
  }
  printf("%d\n", cnt);
}
